translator/translate/otel/receiver/otlp/translator.go

```diff
@@ -16,23 +18,24 @@ import (
 )
 
 const (
-	defaultGrpcEndpoint = "127.0.0.1:4317"
-	defaultHttpEndpoint = "127.0.0.1:4318"
+	defaultGrpcEndpoint           = "127.0.0.1:4317"
+	defaultHttpEndpoint           = "127.0.0.1:4318"
+	defaultAppSignalsGrpcEndpoint = "0.0.0.0:4315"
+	defaultAppSignalsHttpEndpoint = "0.0.0.0:4316"
 )
 
 var (
 	configKeys = map[component.DataType]string{
-		component.DataTypeTraces: common.ConfigKey(common.TracesKey, common.TracesCollectedKey, common.OtlpKey),
+		component.DataTypeTraces:  common.ConfigKey(common.TracesKey, common.TracesCollectedKey),
+		component.DataTypeMetrics: common.ConfigKey(common.LogsKey, common.MetricsCollectedKey),
```

Comment:
Im confused why we introduced `logs -> metrics_collected -> otlp` support in this PR.